telemetry/README.md

```diff
@@ -0,0 +1,23 @@
+# Glean Development Guide
+
+*This is a work in progress!*
+
+## Updating the `metric.yaml` file
+
+Currently, the source of truth for the shared `metrics.yaml` file lives [here](https://gist.github.com/kirill-demtchouk/41a38336b61b938833fe011bda4a2001) as a gist in Kirill's account. We need to figure out a better solution for this long-term but for the sake of getting everyone up to speed on where we are currently.
+
+When this file has been updated, Kirill will notify us and we need to make changes to update it. Here's my current process:
+
+1. Copy the raw JSON of Kirill's `metrics.yaml` [gist](https://gist.github.com/kirill-demtchouk/41a38336b61b938833fe011bda4a2001)
+2. Paste that raw JSON in the `metrics.yaml` file here in elk, replacing everything that was there before
+3. Check git diff to make sure I understand what is changing between the previous version and this one
+4. Add back the things that the shared `metrics.yaml` file removes
+  4a. Specifically, this is the `lifetime: application` and the `no_lint: -baseline` declarations
+    - Without adding the `no_lint: - baseline`, the `build:glean` script will fail
+    - The `lifetime: application` is needed to make these identifiers show up with every ping
+5. Remove the extra things in the shared `metrics.yaml` file that aren't specific to web
```

Comment:
ay caramba

Reply:
![foil](https://github.com/MozillaSocial/elk/assets/1273879/75c267e1-8c0d-4f5d-86bb-6309e9c6fbe1)

Reply:
"There's gotta be a better way! 😭"

Reply:
![giphy-6](https://github.com/MozillaSocial/elk/assets/2893463/f57a0e1a-2168-48d8-9c11-f970d8105158)